src/validate.py

```diff
@@ -144,3 +125,44 @@ def parse_accept_language(header, supported_langs):
 
     # If all else fails, default the language.
     return DEFAULT_LANGUAGE.lang_code
+
+def get_json_files(path):
+
+    files_found = []
+    for root, directories, files in os.walk(path):
+        for file in files:
+            if '.json' in file:
+                files_found.append(os.path.join(root, file))
+
+    return files_found
+
+def get_entries_from_json(path, p_key, s_key):
+
+    with open(path) as json_file:
+        data = json.load(json_file)
+
+    entries = []
+
+    if p_key in data:
+        for values in data.get(p_key):
+             entries.append(values.get(s_key))
+
+    return entries
+
+def get_chapters(file):
+
+    chapters = []
+
+    data = get_entries_from_json(file,'outline','chapters')
+    for list in data:
+        for entry in list:
+            chapters.append(entry.get('title').lower().replace(" ", "-").replace("/",""))
+
+    return chapters
+
+json_files = get_json_files(CONFIG_DIR)
+
+for file in json_files:
```

Comment:
Can we perhaps do this work in the `get_json_files` or `get_chapters` method, instead of spreading it between both there and the module scope? The methods are only used in the one place anyway?

Reply:
have separated the functions based on the unit and type of work so that they remain testable. 

`get_json_files`  walks through the directory structure and provides the list of json files.

since the json file has many segments I tried to keep their retrieval and processing separate as different segments need different processing hence `get_chapters`

let me know if that's desirable.

Reply:
sure, let me try to wrap them up.